src/pyfaf/ureport.py

```diff
@@ -537,11 +550,51 @@ def is_known(ureport, db, return_report=False, opsysrelease_id=None):
     problemplugin = problemtypes[ureport["problem"]["type"]]
     report_hash = problemplugin.hash_ureport(ureport["problem"])
 
-    report = get_report_by_hash(db, report_hash)
+    known_type = []
+
+    # Split allowed types from config
+    if config.has_key('ureport.known') \
+        and config['ureport.known'].strip() != "":
+        known_type = config['ureport.known'].strip().split(" ")
+
+    if len(known_type) > 0 and not valid_known_type(known_type):
+        return None
```

Comment:
Probably better to move all this + validation to pyfaf.utils.parse. Make sure the default is the previous behaviour.